Transformers/ServiceNowFormDataTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace EMS\SubmissionBundle\Transformers;
+
+class ServiceNowFormDataTransformer extends AbstractFormDataTransformer
+{
+    public function transform(): array
+    {
+        $data = $this->form->getData();
+
+        return $this->mapNewFieldTimeFormat($data);
+    }
+
+    private function mapNewFieldTimeFormat(array $data): array
+    {
+        $newFields = $this->getNewFieldTimeFormatValues();
+
+        foreach ($newFields as $k => $v) {
+            $data = $this->arrayReplace($data, $k, $v);
+        }
+
+        return $data;
+    }
+
+    private function getNewFieldTimeFormatValues(): array
+    {
+        $newFields = [];
+
+        $types = [
+            'Symfony\Component\Form\Extension\Core\Type\TimeType'
```

Comment:
This is a hidden dependency. It would be better to use TimeType:class to make the dependency of TimeType explicit. This is much safer if we would refactor on this class.